src/test/java/com/hyunec/cosmicbaseball/acceptancetest/NormalBaseballLv1Test.java

```diff
@@ -1,18 +1,85 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.MockMvc;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest
 class NormalBaseballLv1Test {
-    @DisplayName("타격 결과는 모두 같은 확률을 가집니다.")
+
+    @Autowired
+    private StrikeHandler strikeHandler;
+    @Autowired
+    private BaseballGame baseballGame;
+
     @Test
-    void t1() {
-        throw new RuntimeException("Not yet implemented");
+    @DisplayName("타격 결과는 모두 같은 확률을 가집니다.")
+    void testSwing() {
+
+                // Given
+                int numberOfSwings                = 10000; // 스윙 횟수
+                Map<String, Integer> resultCounts = new HashMap<>();
+                resultCounts.put("스트라이크", 0);
+                resultCounts.put("볼", 0);
+                resultCounts.put("안타", 0);
+
+                // When : 스윙 시뮬레이션
+                for (int i = 0; i < numberOfSwings; i++) {
+                    String result = baseballGame.swing();
+                    resultCounts.put(result, resultCounts.get(result) + 1);
+                }
+
+                // Then
+                int expectedCount = numberOfSwings / 3;
+                int tolerance     = (int) (numberOfSwings * 0.05); // 허용 오차: 5%
+
+                assertTrue(Math.abs(resultCounts.get("스트라이크") - expectedCount) <= tolerance,
+                        "스트라이크 횟수가 허용 오차 범위 내에 있지 않습니다.");
+                assertTrue(Math.abs(resultCounts.get("볼") - expectedCount) <= tolerance,
+                        "볼 횟수가 허용 오차 범위 내에 있지 않습니다.");
+                assertTrue(Math.abs(resultCounts.get("안타") - expectedCount) <= tolerance,
+                        "안타 횟수가 허용 오차 범위 내에 있지 않습니다.");
     }
 
-    @DisplayName("타격 결과는 strike, ball, hit 입니다.")
     @Test
-    void t2() {
-        throw new RuntimeException("Not yet implemented");
+    @DisplayName("타격 결과는 strike, ball, hit 입니다.")
+    void testPlayGame() {
+
+        // Given
+        Map<String, Boolean> resultOccurred = new HashMap<>();
+        resultOccurred.put("스트라이크", false);
+        resultOccurred.put("볼", false);
+        resultOccurred.put("안타", false);
+
+        // When
+        for (int i = 0; i < 100; i++) {
+            String result = baseballGame.swing();
+            if (resultOccurred.containsKey(result)) {
+                resultOccurred.put(result, true);
+            } else {
+                throw new AssertionError("스트라이크, 볼, 안타가 아닌 결과 " + result);
+            }
+        }
+
+        // Then
+        assertTrue(resultOccurred.get("스트라이크"), "스트라이크 결과가 발생하지 않음.");
+        assertTrue(resultOccurred.get("볼"), "볼 결과가 발생하지 않음.");
+        assertTrue(resultOccurred.get("안타"), "안타 결과가 발생하지 않음.");
```

Comment:
assert message 를 별도로 관리하는 방법 좋습니다. 👍
- 하지만 저는 string 을 따로 관리해야되는 귀찮음이 있어 라이브러리에서 제공되는 기본형을 쓰곤 합니다.

---

```suggestion
        assertThat(resultOccurred.get("스트라이크")).isTrue();
        assertThat(resultOccurred.get("볼")).isTrue();
        assertThat(resultOccurred.get("안타")).isTrue();
```

junit 의 assertion 을 그대로 사용하는 지금 방식도 좋지만, 실무에서는 조금 더 확장된 DSL 들을 사용합니다.
- fyi. assertJ, Kluent, awaitility